Dockerfile

```diff
@@ -9,8 +9,19 @@ ARG REPO_INFO
 ARG COMMIT
 RUN CGO_ENABLED=0 GOOS=linux go build -o kong-ingress-controller -ldflags "-s -w -X main.RELEASE=$TAG -X main.COMMIT=$COMMIT -X main.REPO=$REPO_INFO" ./cli/ingress-controller
 
-# Final stage: the running container.
-FROM alpine:3.11
+# Build Alpine image
+FROM alpine:3.11 AS alpine
+ARG TAG
+LABEL name="Kong Ingress Controller" \
+      vendor="Kong" \
+      version="$TAG" \
+      release="1" \
+      url="https://github.com/Kong/kubernetes-ingress-controller" \
+      summary="Kong for Kubernetes Ingress" \
+      description="Use Kong for Kubernetes Ingress. Configure plugins, health checking, load balancing and more in Kong for Kubernetes Services, all using Custom Resource Definitions(CRDs) and Kubernetes-native tooling."
+
+COPY LICENSE /licenses/
```

Comment:
It's a Red Hat requirement for listing on their registries. This PR doesn't push to them, though I suppose I should add that for the release builds given that we'll have the image for it.

Their docs aren't easily navigable and some parts are hidden behind a login wall. That

> Test name: has_licenses
Why? So the end user is aware of the terms and conditions applicable to the software. Including opens source licensing information, if open source components are included in the image.
How? Create a directory named /licenses and include all relevant licensing and/or terms and conditions as text file(s) in that directory.

The closest public equivalent I can find is https://redhat-connect.gitbook.io/partner-guide-for-red-hat-openshift-and-container/program-on-boarding/containers-with-red-hat-universal-base-image-ubi and [the linked examples](https://github.com/RHC4TP/starter/tree/master/Container%20Zone/UBI-Example).

Reply:
I see, thanks for explaining the context to me.

I think it's a more complex problem than it seems: the `LICENSE` file is most probably not enough - licenses such as Apache 2.0 require redistribution of copyright notices (usually in the form of a `NOTICE` file) as well. It may be so that there are no such notices today but they will be added in the future, posing a risk of accidental omission.

To keep the scope manageable, I recommend that we treat satisfying RH's license needs as a separate problem (by creating a separate issue to track sufficient introduction of license data to docker images - @eskibars may probably help here). Any (potentially) half-baked solution here will suppress the check on RH's side but potentially omit something important.